scout/build/individual.py

```diff
@@ -66,7 +67,9 @@ def build_individual(ind):
         raise(PedigreeError("Unknown phenotype: %s" % phenotype))
```

Comment:
Oh, there are also entries for the file names in the docstring here. Better update!

Reply:
?

I don't follow -can you elaborate 

:)

Reply:
I think what I meant was could you please update the docstring for this function with the new options?